ДЗ.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>Седона</title>
+</head>
+
+<body>
+
+<hedar>
+ <nav>
+  <a>
+      <img src="img/welcome.png" width="456.65" height="350.88" alt="Логотип «Седона»">
+    </a>
+   
+  <ul>
+      <li><a href="blank.html">Информация</a></li>
+      <li><a href="photos and videos.html">фото и Видео</a></li>
+      <li><a href="the map of the state.html">Карта штата</a></li>
+      <li><a href="hotels.html">Гостиницы</a></li>
+    </ul>
```

Comment:
Ссылки на страницы лучше прописывай без пробелов используя _ или - вместо них
```suggestion
  <ul>
      <li><a href="blank.html">Информация</a></li>
      <li><a href="photos-and-videos.html">фото и Видео</a></li>
      <li><a href="the_map_of_the_state.html">Карта штата</a></li>
      <li><a href="hotels.html">Гостиницы</a></li>
    </ul>
```
Но совсем хорошо будет одним словом
```suggestion
  <ul>
      <li><a href="blank.html">Информация</a></li>
      <li><a href="media.html">фото и Видео</a></li>
      <li><a href="map.html">Карта штата</a></li>
      <li><a href="hotels.html">Гостиницы</a></li>
    </ul>
```